ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -176,8 +176,8 @@ public static class ValidationRegexes
         public const string CharacterValidation = @"[a-zA-Z \-—,.']+";
         public const string CharacterValidationWithNumbers = @"[0-9a-zA-z \-—,.']+";
         public const string CharacterValidationWithNumbersForwardSlashAndNewLine = @"[0-9a-zA-Z \/\-—,.'\n\r]+";
-        public const string CharacterValidationWithNumbersForwardSlashExtended = @"[0-9a-zA-Z \/\-—,.'`@#&+;:$_()\\\[\]=\*\?]+";
-        public const string CharacterValidationWithNumbersForwardSlashExtendedWithNewLine = @"[0-9a-zA-Z \/\-—,.'`@#&+;:$_()<>\\\[\]=\*\?\n\r]+";
+        public const string CharacterValidationWithNumbersForwardSlashExtended = @"[0-9a-zA-Z \/\-—,.'`@#&+;:$_()\\\[\]=\*\?\""]+";
+        public const string CharacterValidationWithNumbersForwardSlashExtendedWithNewLine = @"[0-9a-zA-Z \/\-—,.'`@#&+;:$_()<>\\\[\]=\*\?\""\n\r]+";
```

Comment:
These strings are infuriating - how have these character lists been determined? e.g. Why does the bottom one contain `<>`? Why are other characters like `{` and `}` excluded?

Were you able to find any documentation on this? Without knowing the answers to these questions it's impossible to know if there are other characters missing.

Reply:
Thanks for this context :-)

I think for now we should avoid addressing the broader problem, but we should make sure that these two are consistent, and that they contain all of the printable basic ASCII characters (i.e. numbers 32-126). I think we already have most of them, so going up to that would at least give these sets a meaning.

Reply:
Sorry just saw this comment about local patient ID - please can you add the wiki link here?